recipe/meta.yaml

```diff
@@ -13,20 +13,20 @@ package:
   name: {{ name }}
   version: {{ version }}
 
-source:
-  url: https://github.com/single-cell-data/TileDB-SOMA/archive/{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+#source:
```

Comment:
This section for source + url + sha256 needs to be uncommented.

Reply:
that was an embarrassingly dumb mistake on my part 😐